### ISSUE TITLE
Needs a rubygems.org release

### ISSUE THREAD

**Issue by elliotcm** [zinc-collective/factory_bot-namespaced_factories#1]
Howdy!  Thanks again for this.

Looks like it needs a rubygems release, though.  Can't use github sources to add dependencies in gemspecs.

**Comment by elliotcm** [comment]
@zspencer I've gone ahead and released this on my own account, let me know if/when you want me to transfer ownership.

https://rubygems.org/gems/factory_bot_namespaced_factories

**Comment by zspencer** [comment]
Oh poop, for whatever reason Github notifications don't actually come to email for me. Thanks for releasing! If you want maintainer rights here, I'm down.

**Comment by elliotcm** [comment]
Sounds good :)

Do you have an email address you'd like me to assign as an owner for the gem?  So you can release to it as well.

**Comment by zspencer** [comment]
You should be able to add me as an owner via my rubygems handle: `zee` (To test this, I invited you to `car`, my silly joke gem I made 9 years ago)

**Comment by elliotcm** [comment]
Perfect.  Before I do, it might be worth renaming it to `factory_bot-namespaced_factories`.  It's just convention, but I believe bundler won't be able to infer the `require` for it otherwise.  See https://guides.rubygems.org/name-your-gem/#mix-underscores-and-dashes-appropriately

**Comment by zspencer** [comment]
@elliotcm  - I pushed a rename to Main; but I can't push to RubyGems because it's too similar to `factory_bot_namespaced_factories`: 

```
rake build
gem push pkg/factory_bot-namespaced_factories-0.1.0.gem
Pushing gem to https://rubygems.org...
There was a problem saving your gem: Name 'factory_bot-namespaced_factories' is too similar to an existing gem named 'factory_bot_namespaced_factories'
```

**Comment by elliotcm** [comment]
@zspencer Sorted :)

**Comment by zspencer** [comment]
Thank you! I've released v0.1.1 which has the appropriate naming, as well as slightly better documentation.
